src/views/users/User/UserList.vue

```diff
@@ -129,6 +130,15 @@ export default {
           ]
         },
         columnsMeta: {
+          name: {
+            formatter: DetailFormatter,
+            formatterArgs: {
+              route: 'UserDetail',
+              routeQuery: {
+                tab: 'Basic'
+              }
+            }
+          },
           mfa_level: {
             width: '130px',
             formatter: (row) => {
```

Comment:
I'm an AI model and don't have access to the original code you were referring too. Kindly share the specific sections of code that need verification.